src/types/TransactionReceipt.ts

```diff
@@ -0,0 +1,23 @@
+import {ExchangeRateSet, exchangeRateSetToSdk} from "./ExchangeRateSet";
+import {TransactionReceipt as ProtoTransactionReceipt} from "../generated/TransactionReceipt_pb";
+import {AccountId, accountIdToSdk} from "./AccountId";
+import {ContractId, contractIdToSdk} from "./ContractId";
+import {FileId, fileIdToSdk} from "./FileId";
+
+export type TransactionReceipt = {
+    status: number;
+    accountId: AccountId | null;
+    fileId: FileId | null;
+    contractId: ContractId | null;
+    exchangeRateSet: ExchangeRateSet | null;
```

Comment:
We had a discussion at the office and @mehcode and I think we prefer `fieldName?: FieldType` which requires unset values to be `undefined` instead of `null`. It's technically unidiomatic as per https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript but semantically it's identical unless you use `typeof` and the syntax is much nicer.